privaterelay/utils.py

```diff
@@ -55,11 +58,37 @@ def get_countries_info_from_lang_and_mapping(
 
 
 def _get_cc_from_request(request: HttpRequest) -> str:
+    """Determine the user's region / country code."""
+
+    log_data: dict[str, str] = {}
+    cdn_region = None
+    region = None
     if "X-Client-Region" in request.headers:
-        return request.headers["X-Client-Region"].upper()
+        cdn_region = region = request.headers["X-Client-Region"].upper()
+        log_data["cdn_region"] = cdn_region
+        log_data["selected"] = "cdn"
```

Comment:
suggestion (non-blocking): `selected` is ambiguous. When I first saw it I expected `selected` to mean "Which region was selected" - e.g., `selected: DE`. Can we change it to `region_selection_method` to be more explicit?

Reply:
is `region_method` OK? I was mostly avoiding `method`, which is ambiguous with HTTP request method (`POST`, `GET`, etc)